adal/src/main/java/com/microsoft/aad/adal/AuthenticationConstants.java

```diff
@@ -477,6 +526,7 @@ private AuthenticationConstants() { }
 
         /** The value for pkeyauth redirect. */
         public static final String PKEYAUTH_REDIRECT = "urn:http-auth:PKeyAuth";
+        public static final String CLIENT_TLS_REDIRECT = "urn:http-auth:PKeyAuth";
```

Comment:
exactly same as previous line

Reply:
CLIENT_TLS_REDIRECT used by other packages. I would like to merge it in this way first, and do a small refactor later to update all files using this variable name. Does this make sense ?